app/models/transaction.rb

```diff
@@ -25,6 +25,15 @@ class Transaction < ApplicationRecord
     inclusion: %w( NEFT Withdraw Deposite )
   )
 
+  #
+  ## Constant
+  #
+  TRANSACTION_TYPES = [
```

Comment:
@babarajay , What is missing in this declaration?

Reply:
I need to freeze the constant.